teuthology/suite/util.py

```diff
@@ -209,6 +209,8 @@ def git_validate_sha1(project, sha1, project_owner='ceph'):
         # kinda specific to knowing git.ceph.com is gitweb
         url = ('http://git.ceph.com/?p=%s.git;a=blob_plain;f=.gitignore;hb=%s'
                % (project, sha1))
+    elif config.ceph_git_base_url in url:
+        url = '/'.join((url, 'commit', sha1))
```

Comment:
The code of the method is making request based on the url of git repo in different services, in particular it uses REST api of GitHub and GitWeb based on the repo url. I can't understand why variable url should have the format which is suggested within the change, and which use cases it can be utilized.